config/manager/default-config/db-statefulset.yaml

```diff
@@ -29,7 +29,7 @@ spec:
           envFrom:
             - secretRef:
                 name: <POSTGRESQL_SECRET>  # will be replaced with 'backstage-psql-secrets-<cr-name>'
-          image: <RELATED_IMAGE_postgresql> # will be replaced with the actual image
+          image: quay.io/fedora/postgresql-15:latest
```

Comment:
Moving the default images to the default configs does not seem to solve the issue as intended. The defaults are still hard-coded although now they are in the configmap.

Also seems to me this does mot make  the operator safer but in opposite: with current implementation if RELATED_IMAGE_postgresql is not provided, the operator will issue an error, while with this PR the defaults get actually

Reply:
No, it is not hardcoded, it is yaml configuration which can be changed by kubectl apply (or so), become configMap 
 and is part of the bundle (along with CSV yaml).
 
 I am not sure I understand your "safer" point. Yes, Admin should specify the default images in the Operator config (as mentioned in Admin guide), if he/she does not it will fail...

Reply:
Not sure I understand the point...
Yes, you can change either <RELATED_IMAGE_backstage> or <RELATED_IMAGE_postgresql> independently but my undrstanding is <RELATED_IMAGE_backstage> is applied for both backstage and init container?
Otherwise we need 3 env variables (backstage, init and postgres). Did I miss something?

Reply:
My point was that the current implementation on `main` could allow someone to use different images between init and main containers, by not setting the `<RELATED_IMAGE_backstage>` placeholder in the ConfigMap (or setting it for main container only and using different image for init container); and this is not desirable right now.
So I am fine with the changes here :p